<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>397</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>AllHail</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="4">
       <widget class="QGraphicsView" name="atlasView"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="1" column="2">
         <widget class="QPushButton" name="rightButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="downButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="leftButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="upButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QGraphicsView" name="previewView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string>Status Bar</string>
   </property>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionNew_Project</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newProjectDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updatePreview(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updatePreview(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addToScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>snapUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>snapDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>snapLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>snapRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateStatusBarText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateStatusBarText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateStatusBarText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateStatusBarText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchFileDialog()</slot>
  <slot>updatePreview(QListWidgetItem*)</slot>
  <slot>addToScene()</slot>
  <slot>snapUp()</slot>
  <slot>snapDown()</slot>
  <slot>snapLeft()</slot>
  <slot>snapRight()</slot>
  <slot>updateStatusBarText()</slot>
  <slot>newProjectDialog()</slot>
 </slots>
</ui>
